packages/web-components/fast-foundation/src/tree-item/tree-item.ts

```diff
@@ -235,10 +247,12 @@ export class TreeItem extends FASTElement {
             return;
         }
 
-        if (!this.expanded) {
+        if (!this.expanded && this.childItemLength() > 0) {
             this.setExpanded(true);
         } else {
-            this.focusNextNode(1);
+            if (this.childItemLength() > 0) {
+                this.enabledChildTreeItems()[0].focus();
```

Comment:
Is it possible that childItemLengty is greater than 0 but that the enabled child tree items is 0?

Reply:
I think this is technically possible and a good case to guard against. I think there was an email where I called this out as a noop ;). We should likely account for this. Good catch @eisenbergeffect!

Reply:
doh! will fix